### DISCUSSION TITLE
how to specify package path in renovate.json if the package inside repo

### DISCUSSION THREAD

**Discussion by hbakhtiyor** [renovatebot/renovate#25606] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

packages/ac/core/package.json

```
{
  "name": "@demo/core",
  "version": "0.0.1",
  "main": "_/index.js",
  "type": "module"
'
  "dependencies": {
    "@demo/lib": "0.2.0",
  }
}
```

packages/ab/lib/package.json

```
{
  "name": "@demo/lib",
  "version": "0.2.0",
  "dependencies": {
    "debug": "4.3.4",
    "uuid": "9.0.0"
  }
}

```

what's the configuration need to set for looking that path for only specific packages instead of npm registry?

### What you tried so far.

nothing

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Your description is too vague to answer. For example it's not clear what you mean by "that path"

**Reply by hbakhtiyor** [reply]
to the packages/ab/lib/
